test/e2e_node/remote/node_e2e.go

```diff
@@ -62,6 +62,10 @@ func (n *NodeE2ERemote) SetupTestPackage(tardir, systemSpecName string) error {
 	requiredBins := []string{"kubelet", "e2e_node.test", "ginkgo", "mounter", "gcp-credential-provider"}
 	for _, bin := range requiredBins {
 		source := filepath.Join(buildOutputDir, bin)
+		if builder.IsBuildForArm64() {
+			source = filepath.Join(buildOutputDir, "linux_arm64", bin)
```

Comment:
is this correct for dockerized mode? On my machine with dockerized build, the build directory ends up being, e.g. 

```
$ build/run.sh make WHAT=cmd/kubelet KUBE_BUILD_PLATFORMS=linux/arm64
$ ls _output/dockerized/bin/linux/arm64/kubelet
_output/dockerized/bin/linux/arm64/kubelet
```

Do we need to add also need to add  `/dockerized/` to the build output directory when it's building in dockerized mode?
I believe `GetK8sBuildOutputDir` assumes non-dockerized build... (xref: https://github.com/kubernetes/kubernetes/blob/master/test/utils/paths.go#L67-L78)

Additionally, we should also ensure this work for non ARM, i.e. dockerized build for amd64, so I don't think we should hardcode `linux_arm64`, but instead rely on the the target build arch flag.

Reply:
Hmm, I tested with this setup and the tests worked as expected: https://github.com/kubernetes/kubernetes/pull/117017#issuecomment-1550529226

Let me double check